modules/configuring-ide.adoc

```diff
@@ -0,0 +1,199 @@
+:_module-type: PROCEDURE
+
+[id="configuring-ide_{context}"]
+= Configuring your IDE
+
+[role='_abstract']
+You can configure some notebook workbenches to get the most out of your data science work.
+
+== Configuring VS Code Server
+
+Incorporating standard database clients into your VS Code Server environment can streamline your workflow, offering a seamless interface for managing databases. You can incorporate different Database Management System (DBMS) clients into your Code Server workbench through built-in VS Code extensions or by creating a custom notebook image using Dockerfile.
```

Comment:
Should this be "...using a Dockerfile"?

Reply:
Good spot, will update here and line 80.